packages/chat/components/chat/src/chat.scss

```diff
@@ -0,0 +1,272 @@
+/**
+ * Copyright IBM Corp. 2023, 2024
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+@use '../../../../../globals/scss/vars' as *;
+@use '@carbon/styles/scss/theme' as *;
+:host(#{$c4ai-prefix}-chat) {
+  position: relative;
+  display: block;
+  block-size: 320px;
+  font-family: 'IBM Plex Sans', sans-serif;
+  min-block-size: 320px;
+
+  .chat-container {
```

Comment:
In Carbon we follow the [CSS BEM (Block Element Modifier) naming convention](https://getbem.com/introduction/) and use prefixes in the classnames: 

Example here of the text-input component: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/styles/scss/themes/expressive/components/_text-input.scss#L26

Reply:
done, but should the prefix be added to every single class I have in my css or just the container?

Reply:
this should be done for every single class - you can use the carbon-for-ibm-dotcom repo as a reference: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/packages/carbon-web-components/src/components/text-input/text-input.scss